### DISCUSSION TITLE
Restore backup from different rancher cluster

### DISCUSSION THREAD

**Discussion by AliyevH** [longhorn/longhorn#5423] [Q&A]
Hi. Is it possible to take backup PVC in one rancher cluster, and restore it in another one?
If yes, how to do it? 
Does cli tool exists for longhorn?

**Comment by weizhe0422** [comment]
I don't know what the purpose of your second cluster is. IIUC, I think what you say is workable.

You can set same backup target for both cluster. Then you can restore or create DR volume with the latest backup in the second cluster.

You can also refer to below links to get the detail information.
[Backup and Restore](https://longhorn.io/docs/1.4.0/snapshots-and-backups/backup-and-restore/)
[Disaster Recovery Volumes](https://longhorn.io/docs/1.4.0/snapshots-and-backups/setup-disaster-recovery-volumes/)
